CookuPots/Screens/Categories/CategoriesVC.swift

```diff
@@ -0,0 +1,255 @@
+//
+//  CategoriesVC.swift
+//  CookuPots
+//
+//  Created by Sebulla on 23/03/2022.
+//
+
+import UIKit
+
+class CategoriesVC: UICollectionViewController {
+    
+    let searchBar = UISearchBar()
+    
+    static let categoryHeaderId = "categoryHeaderId"
+    static let recomendedHeaderId = "recomendedHeaderId"
+    let headerId = "headerId"
+    let recommendedId = "recommendedId"
```

Comment:
1. these can all be `private`

2. please delete unused code (I think you don't need recommendHeaderID)?